dev/deps/spark-deps-hadoop-3.2

```diff
@@ -50,10 +52,11 @@ curator-client-2.13.0.jar
 curator-framework-2.13.0.jar
 curator-recipes-2.13.0.jar
 datanucleus-api-jdo-3.2.6.jar
-datanucleus-core-3.2.10.jar
+datanucleus-core-4.1.17.jar
 datanucleus-rdbms-3.2.9.jar
 derby-10.12.1.1.jar
 dnsjava-2.1.7.jar
+dropwizard-metrics-hadoop-metrics2-reporter-0.1.2.jar
```

Comment:
These and other new dependencies appear to be ALv2 licensed, as expected as they're coming from Hive, so OK.